### PR TITLE
Teach bazelisk.py about .bazeliskrc

### PR DESCRIPTION
**Pull request by jwnimmer-tri**
This allows for suggesting (but not requiring) a bazel version using the bazeliskrc dotfile.

### CODE REVIEW

**Comment by jwnimmer-tri** [comment]
Hi team. Is there anything I can do to help move this forward?  I just fixed the merge conflict errors with a rebase.

**Comment by jwnimmer-tri** [comment]
Hi team. Is there anything I can do to help move this forward?

**Comment by jwnimmer-tri** [comment]
Hi team. Is there anything I can do to help move this forward?  All tests are passing.

**Comment by albertocavalcante** [comment]
@jwnimmer-tri just wondering - is this still relevant? (I didn't take a look yet)

**Comment by jwnimmer-tri** [comment]
Yes, I believe so.

I just rebased and re-tested locally and all tests passed.

File: bazelisk.py [hunk 1]
```diff
@@ -68,6 +68,31 @@
 BAZEL_UPSTREAM = "bazelbuild"
 
 
+def get_env_or_config(name, default=None):
```
Comment (fweikert) [inline bazelisk.py:71]: Ideally we'd only read the actual file once and cache the results.

Comment (jwnimmer-tri) [reply bazelisk.py]: My thought process was that `bazelisk.py` should prioritize simplicity and portability over raw speed (users who want raw speed should be using the native version), but anyway I pushed a second commit to add the memoization.

**Comment by jwnimmer-tri** [comment]
This should be ready for re-review now.

**Comment by jwnimmer-tri** [comment]
The label "awaiting-user-response" is no longer accurate -- I have responded.

**Review by fweikert — APPROVED** [review]

**Comment by fweikert** [comment]
Thank you!

**Comment by jam3120** [comment]
Gracias

**Comment by jam3120** [comment]
``

**Pull request merged**